pkg/operator/ceph/operator.go

```diff
@@ -143,8 +143,17 @@ func (o *Operator) Run() error {
 		logger.Infof("rook-provisioner %s started using %s flex vendor dir", name, vendor)
 	}
 
+	var namespaceToWatch string
+	if os.Getenv("ROOK_CURRENT_NAMESPACE_ONLY") == "true" {
```

Comment:
Is that the reason why everything is moved to be in `rook-ceph` namespace again as it was originally?

Reply:
This setting is to allow running multiple operators where each operator only watches for the CRDs in its own namespace instead of all namespaces. As a side effect, yes, the default now is to run the operator is the `rook-ceph` namespace with the cluster.